src/Neo/SmartContract/Native/StdLib.cs

```diff
@@ -169,12 +169,41 @@ public static string Base58CheckEncode([MaxLength(MaxInputLength)] byte[] data)
         /// </summary>
         /// <param name="s">The base58 <see cref="string"/>.</param>
         /// <returns>The decoded byte array.</returns>
-        [ContractMethod(CpuFee = 1 << 16)]
+        [ContractMethod(Hardfork.HF_Echidna, CpuFee = 1 << 16)]
         public static byte[] Base58CheckDecode([MaxLength(MaxInputLength)] string s)
```

Comment:
Is it a typo? I suppose we should keep `Base58CheckDecode` method unchanged.

Reply:
my fault,,,,,,, wrong place.